experimental/devices/ads1x15/ads1x15.go

```diff
@@ -0,0 +1,392 @@
+package ads1x15
+
+import (
+	"encoding/binary"
+	"errors"
+	"math"
+	"sync"
+	"time"
+
+	"periph.io/x/periph/conn/physic"
+	"periph.io/x/periph/conn/pin"
+
+	"fmt"
+
+	"periph.io/x/periph/conn/i2c"
+)
+
+const (
+
+	// ADS1x15DefaultAddress is the default I2C address for the ADS1x15 components
+	ADS1x15DefaultAddress = 0x48
+
+	ads1x15PointerConversion    = 0x00
+	ads1x15PointerConfig        = 0x01
+	ads1x15PointerLowThreshold  = 0x02
+	ads1x15PointerHighThreshold = 0x03
+	// Write: Set to start a single-conversion
+	ads1x15ConfigOsSingle       = 0x8000
+	ads1x15ConfigMuxOffset      = 12
+	ads1x15ConfigModeContinuous = 0x0000
+	//Single shoot mode
+	ads1x15ConfigModeSingle = 0x0100
+
+	ads1x15ConfigCompWindow      = 0x0010
+	ads1x15ConfigCompAactiveHigh = 0x0008
+	ads1x15ConfigCompLatching    = 0x0004
+	ads1x15ConfigCompQueDisable  = 0x0003
+
+	Channel0 = 0
```

Comment:
IMO these constants are not useful, remove.

*Unless*

You do:
```
// Channel is the DAC sampling to return.
//
// It is either the raw value of one pin, or the delta between two pins.
type Channel int

const (
  // Reads an individual DAC channel.
  Channel0 = 4
  Channel1 = 5
  Channel2 = 6
  Channel0 = 7
  // Reads the difference between two channels.
  Channel0Minus1 = 0
  Channel1Minus3 = 1
  Channel1Minus3 = 2
  Channel1Minus3 = 3
)
```

And then you can remove the function PinForDifferenceOfChannels() and inline prepareQuery().